src/app/containers/PodcastPromo/index.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useContext } from 'react';
+import path from 'ramda/src/path';
+import { string } from 'prop-types';
+import PodcastPromo from '@bbc/psammead-podcast-promo';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import ImageWithPlaceholder from '#containers/ImageWithPlaceholder';
+
+const Promo = ({
+  title,
+  brandTitle,
+  brandDescription,
+  imageSrc,
+  imageAlt,
+  linkHref,
+  linkText,
+}) => {
+  const { podcastPromo, script, service } = useContext(ServiceContext);
+  const podcastPromoTitle = title || path(['title'], podcastPromo);
+  const podcastBrandTitle = brandTitle || path(['brandTitle'], podcastPromo);
+  const description =
+    brandDescription || path(['brandDescription'], podcastPromo);
+  const img = imageSrc || path(['image', 'src'], podcastPromo);
+  const alt = imageAlt || path(['image', 'alt'], podcastPromo);
+  const url = linkHref || path(['linkLabel', 'href'], podcastPromo);
+  const label = linkText || path(['linkLabel', 'text'], podcastPromo);
+
+  const showPromo = [
+    podcastBrandTitle,
+    podcastPromoTitle,
+    description,
+    img,
+    alt,
+    url,
+    label,
+  ].every(Boolean);
+  if (!showPromo) {
+    return null;
+  }
+
+  return (
+    <div>
+      <PodcastPromo
+        script={script}
+        service={service}
+        role="region"
+        aria-labelledby="podcast-promo"
+      >
+        <PodcastPromo.Title id="podcast-promo">
+          {podcastPromoTitle}
+        </PodcastPromo.Title>
+        <PodcastPromo.Card>
+          <PodcastPromo.Card.ImageWrapper>
+            <ImageWithPlaceholder
+              src={img}
```

Comment:
Don't know if we have other resolutions for this image, but we could consider generating a `srcSet` for this if so - was one of the original AC @ https://github.com/bbc/psammead/issues/3987

> should be a responsive image using correct srcset and sizes attributes

Reply:
Needs appropriate sizes attribute too